packages/terra-i18n/docs/AggregateTranslations

```diff
@@ -0,0 +1,59 @@
+# Aggregating Translations
+The terra-i18n packages provides internationalization to React component to load translations and locale data on demand. To successfully render internationalized component, all translation information must be provided as a single file for each locale.
```

Comment:
Maybe change to (asterisks emphasize what i change): The terra-i18n packages provides internationalization *for a* React component to load translations and locale data on demand. To successfully render *an* internationalized component, all translation information must be provided as a single file for each locale.